src/main/cpp/subsystems/Intake.cpp

```diff
@@ -21,13 +21,11 @@ Intake::Intake(Flywheel& flywheel) : m_flywheel(flywheel) {
     m_conveyorMotor.SetSmartCurrentLimit(80);
 }
 
-void Intake::Deploy() { m_arm.Set(frc::DoubleSolenoid::kForward); }
+void Intake::Deploy() { m_arm.Set(kArmForward); }
 
-void Intake::Stow() { m_arm.Set(frc::DoubleSolenoid::kReverse); }
+void Intake::Stow() { m_arm.Set(kArmReverse); }
```

Comment:
Call Set() with true or false here, and we'll figure out what that corresponds to IRL.